src/app/components/addFinancial/addFinancial.component.ts

```diff
@@ -18,23 +19,35 @@ export class AddFinancialComponent implements OnInit {
                 libelle: [
                     '', Validators.required
                 ],
-                rentree: [
-                    '', Validators.required
+                inOrOut: [
+                    '1', Validators.required
                 ],
-                sortie: ['', Validators.required]
+                amount: [, Validators.required]
             });
     }
 
     ngOnInit() {}
 
     add() {
 
+
+
+        const data = {
+                date: this.form.value.date,
+                libelle : this.form.value.libelle,
+                enter : this.form.value.amount                                         
+        };
+    
         this
             .bilanService
-            .insert(this.formMembre.value)
-            .subscribe(membre => {});
-
+            .insert(data)
+            .subscribe(membre => {
+                this
```

Comment:
cs : `this.dialogRef.close();`

Reply:
you mean that it should be in a single line ?

Reply:
yep